README.md

```diff
@@ -39,9 +39,41 @@ View dependency in package.json:
 Search in tree:
 ![Search_In_Tree](resources/readme/gifs/search.gif)
 
-To filter scan results, click on the Filter ![Filter](resources/readme/filter.png) button.
+To filter scan results, click on the Filter ![Filter](resources/readme/filter.png) button:
 ![Filter](resources/readme/gifs/filter.gif)
 
+### Proxy
+
+To use HTTP/S proxy:
+
+1. Go to Preferences --> Settings --> Application --> Proxy
+1. Set the proxy URL under 'Proxy'.
+1. Make sure 'Proxy Support' is 'override' or 'on'.
+
+- As alternative, you can use the http_proxy and https_proxy environment variables.
```

Comment:
As alternative --> Alternatively

Shouldn't the env vars be upper case? (HTTP_PROXY and HTTPS_PROXY).

Reply:
Its case insensitive. Changing to uppercase to be more clear.